### PR TITLE
[ES] Add `Client/Options/Keyboard_bindings`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+| Nombre | Por defecto |
+| :-- | :-- |
+| Alternar límite de velocidad de fotogramas | `F7` |
+| Alternar chat | `F8` |
+| Alternar chat extendido | `F9` |
+| Guardar captura de pantalla | `F12` |
+| Aumentar volumen | `Flecha arriba` |
+| Disminuir volumen | `Flecha abajo` |
+| Deshabilitar botones del mouse | `F10` |
+| Tecla de jefe | `Insert` |
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:73]:
```suggestion
| Tecla jefe | `Insertar` |
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+- Explicación de la `Tecla de jefe`: [Tecla de jefe](https://en.wikipedia.org/wiki/Boss_key)
+  - Cuando se usa, osu! se silenciará y ocultará la ventana en la barra de herramientas.
+  - Para deshabilitar la tecla de jefe, haga clic en el icono de osu! en la barra de tareas y se abrirá el osu!client.
+
+## Editor
+
+| Nombre | Por defecto |
+| :-- | :-- |
+| Herramienta de selección | `1` |
+| Herramienta normal | `2` |
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:86]:
```suggestion
| Herramienta Normal | `2` |
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+| Herramienta del deslizador | `3` |
+| Herramienta de la ruleta | `4` |
+| Alternar nuevo combo | `Q` |
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:89]:
```suggestion
| Alternar Nuevo combo | `Q` |
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+| Alternar silbato | `W` |
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:90]:
```suggestion
| Alternar Silbato | `W` |
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+| Alternar finalizar | `E` |
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:91]:
```suggestion
| Alternar Platillos | `E` |
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+| Alternar aplauso | `R` |
+| Alternar ajuste de la rejilla | `T` |
+| Alternar ajuste de distancia | `Y` |
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:94]:
```suggestion
| Alternar Medidor de distancia | `Y` |
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+| :-- | :-- |
+| Herramienta de selección | `1` |
+| Herramienta normal | `2` |
+| Herramienta del deslizador | `3` |
+| Herramienta de la ruleta | `4` |
+| Alternar nuevo combo | `Q` |
+| Alternar silbato | `W` |
+| Alternar finalizar | `E` |
+| Alternar aplauso | `R` |
+| Alternar ajuste de la rejilla | `T` |
```
[the remark this answers is not in the document]
Comment (agatemosu) [reply wiki/Client/Options/Keyboard_bindings/es.md:93]: It is translated like this :/

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
+# Atajos del teclado
+
+Para acceder a este cuadro de diálogo, abra las [Opciones](/wiki/Client/Options), escriba `atajos` y luego haga clic en `Atajos del teclado`.
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:3]:
```suggestion
Para acceder a este cuadro de diálogo, abra las [opciones](/wiki/Client/Options), escriba `atajos` y luego haga clic en `Atajos del teclado`.
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+| Alternar chat extendido | `F9` |
+| Guardar captura de pantalla | `F12` |
+| Aumentar volumen | `Flecha arriba` |
+| Disminuir volumen | `Flecha abajo` |
+| Deshabilitar botones del ratón | `F10` |
+| Tecla jefe | `Insertar` |
+
+**Notas:**
+
+- Explicación de la `Tecla jefe`: [Tecla jefe](https://en.wikipedia.org/wiki/Boss_key)
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:77]:
```suggestion
- Explicación de la `Tecla jefe`: [Boss key](https://en.wikipedia.org/wiki/Boss_key)
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+| Alternar Nuevo combo | `Q` |
+| Alternar Silbato | `W` |
+| Alternar Platillos | `E` |
+| Alternar Aplauso | `R` |
+| Alternar Imantado angular | `T` |
+| Alternar Medidor de distancia | `Y` |
+| Alternar Bloqueo de notas | `L` |
+| Empujar a la izquierda | `J` |
+| Empujar a la derecha | `K` |
+| Alternar ayuda | `H` |
```
Comment (agatemosu) [inline wiki/Client/Options/Keyboard_bindings/es.md:98]:
```suggestion
| Alternar Ayuda | `H` |
```

File: wiki/Client/Options/Keyboard_bindings/es.md [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+| Spun Out | `C` |
+| Auto | `V` |
+
+**Notas:**
+
+- Por defecto, estos atajos están posicionados de la misma manera que se posicionan los iconos de los mods.
+- Es posible que deba presionar la tecla dos veces para acceder a los iconos de los mods ocultos debajo de ellos.
+  - Por ejemplo, deberá presionar `S` dos veces para acceder a Perfect.
+
+## Restablecer todas las asignaciones
```
[the remark this answers is not in the document]
Comment (UnknownOsu) [reply wiki/Client/Options/Keyboard_bindings/es.md:131]: [Así sale en el juego](https://i.imgur.com/BWybQq7.jpeg)

**Review by agatemosu — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**